wazo_calld/plugins/switchboards/stasis.py

```diff
@@ -46,15 +46,20 @@ def notify_all_switchboard_held(self):
             self._notifier.held_calls(switchboard['tenant_uuid'], switchboard['uuid'], held_calls)
 
     def stasis_start(self, event_objects, event):
-        if len(event['args']) < 2:
-            return
-        # app_instance = event['args'][0]
-        if event['args'][1] == 'switchboard_queue':
-            self._stasis_start_queue(event_objects, event)
-        elif event['args'][1] == 'switchboard_answer':
-            self._stasis_start_answer(event_objects, event)
-        elif event['args'][1] == 'switchboard_unhold':
-            self._stasis_start_answer_held(event_objects, event)
+        try:
+            if len(event['args']) < 2:
+                raise Exception('not enough arguments')
+
+            # app_instance = event['args'][0]
+            if event['args'][1] == 'switchboard_queue':
+                self._stasis_start_queue(event_objects, event)
+            elif event['args'][1] == 'switchboard_answer':
+                self._stasis_start_answer(event_objects, event)
+            elif event['args'][1] == 'switchboard_unhold':
+                self._stasis_start_answer_held(event_objects, event)
+        except Exception:
```

Comment:
Is it possible to use more specific error? RequestException or HTTPError?

Reply:
I want to catch pretty much everything here.

If an exception is raised during the stasis call handling the call will be stuck in stasis until Asterisk is restarted or the caller gets tired and hangup.

If anything bad happen planned or not the call should be sent back to the dialplan.

Reply:
Ok :ok_hand: